tidy3d/components/geometry.py

```diff
@@ -35,8 +35,6 @@
     TRIMESH_AVAILABLE = False
 
 try:
-    import networkx  # pylint:disable=unused-import
-    import rtree  # pylint:disable=unused-import
```

Comment:
Without these module imports we won't detect their existence and we'll error out later without further information to the user.

Reply:
thanks for noticing this, actually ruff did this automatically when I added the `--fix` option but I didn't catch it. added them back